### DISCUSSION TITLE
Preferred LES model for flows around an immersed body

### DISCUSSION THREAD

**Discussion by gauravity30** [xcompact3d/Incompact3d#278] [Q&A] [answered]
Hello,
I planning to do LES for flows around immersed body for e.g. a flow around a sphere. Some reference articles use Dynamic Smagorinsky Model. Is the recommended ILES model suitable for such flows ? 

I understand for enabling the ILES model, we need to set
```
ilesmod = 1
jles = 4
smagcst = 0.14
```

Do we need to set any other parameter for the use of ILES for the required flow ? 

Thanks
Gaurav

**Comment by slaizet** [comment, marked as answer]
Hi Gaurav,
We advise our user to use ILES instead of explicit LES models (explicit LES models in Xcompact3d are only implemented for comparison purposes). As explained in our publications, our ILES approach relies on two parameters:
```
! Dissipation control for ILES when ilesmod=1 and jLES=4
nu0nu = 63             ! Ratio between hyperviscosity/viscosity at nu
cnu = 0.22               ! Ratio between hypervisvosity at k_m=2/3pi and k_c= pi
```
`nu0nu` is the amount of dissipation at the cut off wavenumber and `cnu` is the parameter to decide from which wavenumber you want to add numerical dissipation. I suggest you have a read at our publication on our ILES method and you can get familiar with the ILES with the TGV case, see `input_ILES_Re5000.i3d`.

Best,

Sylvain

**Reply by gauravity30** [reply]
Thanks for the answer ! This satisfies my query.
